backend/lambdas/api/authorizer/authorizer/handlers.py

```diff
@@ -29,6 +29,8 @@
 # By default give new users access to scan the demo repo
 DEFAULT_SCOPE = json.loads(os.environ.get("ARTEMIS_DEFAULT_SCOPE", "[]"))
 
+EMAIL_DOMAIN_ALIASES = os.environ.get("EMAIL_DOMAIN_ALIASES", "[]")
```

Comment:
I think the `json.loads()` should happen here (like it does with `DEFAULT_SCOPE` above) rather than down in the `_get_update_or_create_user` method.

Reply:
Fair enough, done